packages/flutter/lib/src/painting/_load_asset_manifest_web.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:js_interop';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+
+@JS('window')
+external _DomWindow get _domWindow;
+
+@JS()
+@staticInterop
+class _DomWindow {}
+
+extension _DomWindowExtension on _DomWindow {
+  @JS('_flutter_assetManifestBytes')
+  external List<int>? get _assetManifestAsByteList;
+}
+
+Future<AssetManifest>? _precachedAssetManifest;
+
+/// Loads the contents of the asset manifest generated at build time.
+///
+// For most web apps, the generated entry point code includes code that writes
+// the asset manifest's contents to a JS property on the global window object.
+// This implementation reads that property.
+Future<AssetManifest> loadAssetManifest(AssetBundle bundle) {
+  if (_precachedAssetManifest != null) {
+    return _precachedAssetManifest!;
+  }
+  final List<int>? assetManifestAsByteList = _domWindow._assetManifestAsByteList;
+  if (assetManifestAsByteList != null) {
+    final ByteData assetManifestByteData = ByteData.view(
+      Uint8List.fromList(assetManifestAsByteList)
+      .buffer
```

Comment:
I think this can be Uint8List.fromList(..).buffer.asByteData()

Reply:
Addressed with commit [+readability to construction of assetManifestByteData](https://github.com/flutter/flutter/pull/130419/commits/c0a81103196a017db3f970eb1950d5b022b4aaa7)